unit-tests/dds/test-stream-sensor-bridge.py

```diff
@@ -0,0 +1,341 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+import pyrealdds as dds
+from rspy import log, test
+dds.debug( True )
+from time import sleep
+
+participant = dds.participant()
+participant.init( 123, "test-stream-sensor-bridge" )
+
+# set up a server device with a bridge
+import d435i
+device_server = dds.device_server( participant, d435i.device_info.topic_root )
+servers = {}
+for server in d435i.build_streams():
+    servers[server.name()] = server
+bridge = dds.stream_sensor_bridge()
+server_list = list( servers.values() )
+bridge.init( server_list )
+device_server.init( server_list, d435i.build_options(), d435i.get_extrinsics() )
+active_sensors = dict()  # sensors that are open; name -> active profiles
+def on_start_sensor( sensor_name, profiles ):
```

Comment:
Please separate function definitions with blank lines.
Can also define elswhere in the file, embedding them here interupts the reading flow

Reply:
They go together with the bridge, that's why they're in same section: these are bridge callbacks.
But OK, I'll add blank lines.